### PR TITLE
perf: update sample to use new API of teamsfx SDK

### PR DESCRIPTION
**Pull request by SLdragon**
Update samples to use new API of latest teamsfx SDK, mainly focus on remove teamsfx class

### CODE REVIEW

**Review by tecton — APPROVED** [review]

File: todo-list-with-Azure-backend/api/todo/index.js [hunk 1]
```diff
@@ -4,9 +4,17 @@
 const {
     TeamsFx,
     getTediousConnectionConfig,
+    OnBehalfOfUserCredential
 } = require("@microsoft/teamsfx");
 const { Connection, Request } = require('tedious');
 
+const oboAuthConfig = {
```
Comment (formulahendry) [inline todo-list-with-Azure-backend/api/todo/index.js:11]: `const oboAuthConfig: OnBehalfOfCredentialAuthConfig`?

File: todo-list-with-Azure-backend/tabs/src/components/Tab.js [hunk 2]
```diff
@@ -31,20 +31,23 @@ class Tab extends React.Component {
   }
 
   async initTeamsFx() {
-    const teamsfx = new TeamsFx();
-    // Get the user info from access token
-    const userInfo = await teamsfx.getUserInfo();
+    const authConfig = {
```
Comment (formulahendry) [inline todo-list-with-Azure-backend/tabs/src/components/Tab.js:34]: Does it have type?

**Review by yiqing-zhao — APPROVED** [review]

File: todo-list-with-Azure-backend/api/todo/index.js (hunk 1 above)
Comment (SLdragon) [reply todo-list-with-Azure-backend/api/todo/index.js:11]: This is not typescript but js file

File: todo-list-with-Azure-backend/tabs/src/components/Tab.js (hunk 2 above)
Comment (SLdragon) [reply todo-list-with-Azure-backend/tabs/src/components/Tab.js:34]: Same above

**Review by formulahendry — APPROVED** [review]

**Pull request merged**